src/templates/pt/2019/error.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base/2019/error.html" %}
+
+{% block unknown_error %}Erro desconhecido{% endblock %}
+
+{% 
+  set localizedStatusCode = {
+    400: "Solicitação Incorreta",
+    404: "Página Não Encontrada",
+    500: "Erro interno do servidor",
+    502: "Erro temporário do servidor"
+  }
+%}
+{% macro expandedTitle(errorTitle) %}Error: {{ errorTitle }}{% endmacro %}
+{%
+  set localizedErrorMessage = {
+    "Unsupported year requested": "Ano solicitado não suportado",
+    "Unsupported chapter requested": "Capítulo solicitado não suportado",
+    "Unsupported language requested": "Idioma solicitado não suportado",
+    "Not Found": "O URL solicitado não foi encontrado no servidor. Se você inseriu o URL manualmente, verifique a ortografia e tente novamente."
+  }
+%}
```

Comment:
Add new line to be consistent with English:

```suggestion
%}

```

Reply:
for me I’m running with the spaces, however, I have no idea why it’s always out of space, but I’ll analyze whenever possible